ToyBox/classes/MainUI/PartyEditor/BrainEditor.cs

```diff
@@ -4,15 +4,21 @@
 using Kingmaker.EntitySystem.Entities;
 using ModKit;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using static ModKit.UI;
 
 namespace ToyBox {
     public partial class PartyEditor {
         public static Browser<AiAction, BlueprintAiAction> browser = new(true, 600, true, true);
         public static Browser<Consideration, Consideration> browser2 = new(true, 600, true, true);
-        public static Browser<Consideration, Consideration> browser3 = new(true, 15, true, true);
+        public static Dictionary<BlueprintAiAction, Browser<Consideration, Consideration>> browsers3;
+        public static UnitEntityData lastCaller;
         public static void OnBrainGUI(UnitEntityData ch) {
+            if (ch != lastCaller) {
+                lastCaller = ch;
+                browsers3 = new();
```

Comment:
I thought the whole point of having an instance was to keep state around.  Why are we throwing it out?